docs/conventions.md

```diff
@@ -54,3 +54,35 @@ Due to privacy reasons, the exact care site and location values are not mapped t
 ### CDM Source
 The name and location of the dataset are not captured in the `cdm_source` table.
 A record is created in this table, with the ETL date, source extraction date, ETL reference, vocab version and OMOP CDM version.
+
+### Death date
```

Comment:
Just wondering: do we want to mention mapping to the observation table as well?

Reply:
No. Upon rereading the OMOP conventions on the death table, I do not think this is required.
> This field is the death date to be used in analysis, as determined by the ETL logic. Any additional information about a Person’s death is stored in the OBSERVATION table with the concept_id 4306655 or in the CONDITION_OCCURRENCE .
https://ohdsi.github.io/CommonDataModel/cdm60.html#PERSON

Reply:
But of course it will not hurt to leave it in for AB.